css/build.css

```
@@ -115,4 +115,77 @@
 
 .close-icon {
   width: 40%;
+}
+
+/*
+  This part of the code needed for support .fa-times-thin class in users who manage to update the LFD component
```

Use this copy. Split into multiple lines as necessary.

```suggestion
  This part of the code is needed for supporting .fa-times-thin class in users who updated to use UNICODE icons instead of SVG icons.
```

---

@tonytlwu Updated the comment